Makefile

```diff
@@ -1,5 +1,11 @@
 run-tests:
 	pipenv run pytest test_pipenv.py
+run-tox:
+	pipenv run tox
```

Comment:
I'm not sure if it's clever to use the module-under-test to actually run the tests. However, on the other side who is going to ensure that `tox` is actually installed ...